modules/ROOT/pages/import-api-task.adoc

```diff
@@ -6,10 +6,10 @@ endif::[]
 You can develop an API for and export it from API Manager for future use or for sharing. The exported API can be imported using this procedure. To perform this task, you need to have a ZIP file on your file system containing an API object exported from API Manager.
 
 . After logging into Anypoint Platform, click API Manager.
-. In API Administration, select Manager API > Import API from the Desktop.
+. In API Administration, select *Manager API > Import API from the Deskto*p.
 . Browse to and select the file ZIP file that contains the exported API instance to import.
 . Click Import.
-
+. If external IdPs are configured for your business organization or environment, select the IdP that you want to use to authenticate access to this API from registered client applications.
```

Comment:
some nits:
* Let's use Client Providers instead of IdPs
* Client Providers are not configurable for business groups
* Authenticate -> Authorize